test/stream-monitor/stream_sources/amqp_source.py

```diff
@@ -246,15 +349,15 @@ def process_tracked_record(self, tracked_record):
     def match_any_event(self, description=None, min=1, max=1):
         if description is None:
             description = "match-any(rk={},min={},max={}".format(None, min, max)
-        m = _AMQPMatcher(route_key=None, description=description, min=min, max=max)
+        m = _AMQPMatcher(self._logs, route_key=None, description=description, min=min, max=max)
         self._add_matcher(m)
 
-    def match_on_routekey(self, description=None, routing_key=None, min=1, max=1):
+    def match_on_routekey(self, description, routing_key=None, min=1, max=1, validation_block=None, match_CB=None):
```

Comment:
Ah. Actually I need to fix test_amqp_heartbeat.py (forcing the use of description was an important step -- otherwise it is impossible to tell what is matching :( )